docs/environment-docs.yml

```
@@ -3,12 +3,12 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.12
+  - python=3.13
```

It seems the RTD build doesn't go passed creating this env, can you try undoing this change?

```suggestion
  - python=3.12
```